### PR TITLE
Little flag to filter updates in Furyfile

### PR DESCRIPTION
**Pull request by nutellinoit**
Hi team!

I added a little flag on the furyctl vendor command ~`--filter`~ `--prefix` to add a prefix on the modules i want to update, eg: ~`furyctl vendor --filter dr/velero`~  `furyctl vendor --prefix dr/velero`.

This can useful when we do stack upgrades.

### CODE REVIEW

**Comment by phisco** [comment]
Is hasprefix enough? What about regexes? Mainly to be able to do an "or" if needed as I don't think "something/.*/whatever" would be a realistic use case in favor of regexes.

Otherwise LGTM

**Comment by angelbarrera92** [comment]
> Is hasprefix enough? What about regexes? Mainly to be able to do an "or" if needed as I don't think "something/.*/whatever" would be a realistic use case in favor of regexes.
> 
> Otherwise LGTM

Agree with Phillipe!

**Comment by nutellinoit** [comment]
I would like to start "easy", in my day to day for this type of activity i don't need regex capabilities, i need only to reduce furyctl vendor scope when updating vendor folder.

**Comment by angelbarrera92** [comment]
> I would like to start "easy", in my day to day for this type of activity i don't need regex capabilities, i need only to reduce furyctl vendor scope when updating vendor folder.

In this case, I suggest renaming the flag. Instead of `f` | `filter`, something like `prefix`. Then we can implement the regex feature later using the `f` | `filter` flag. 

What do you think?

**Comment by nutellinoit** [comment]
> > I would like to start "easy", in my day to day for this type of activity i don't need regex capabilities, i need only to reduce furyctl vendor scope when updating vendor folder.
> 
> In this case, I suggest renaming the flag. Instead of `f` | `filter`, something like `prefix`. Then we can implement the regex feature later using the `f` | `filter` flag.
> 
> What do you think?

Sure @angelbarrera92 , i will do it now

**Comment by nutellinoit** [comment]
Done, instead of filter i added `--prefix` with the `-P` short notation

**Comment by angelbarrera92** [comment]
> Done, instead of filter i added `--prefix` with the `-P` short notation

Can you take a look at the linter? https://ci.sighup.io/sighupio/furyctl/159/1/3

**Comment by nutellinoit** [comment]
> > Done, instead of filter i added `--prefix` with the `-P` short notation
> 
> Can you take a look at the linter? https://ci.sighup.io/sighupio/furyctl/159/1/3

Done!

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

File: cmd/config.go [hunk 1]
```diff
@@ -89,20 +89,38 @@ func (f *Furyconf) Validate() error {
...
+		}
 	}
 	for _, v := range f.Bases {
 		v.kind = "katalog"
-		pkgs = append(pkgs, v)
+		if len(prefix) > 0 {
+			if strings.HasPrefix(v.Name, prefix) {
+				pkgs = append(pkgs, v)
+			}
+		} else {
```
Comment (Rampage1xx) [inline cmd/config.go:121]: same issue

**Review by Rampage1xx — COMMENTED** [review]
my feedbacks

**Comment by Rampage1xx** [comment]
LGTM

**Review by ralgozino — APPROVED** [review]
lgtm

**Comment by angelbarrera92** [comment]
Released at v0.2.5 :)

**Pull request merged**